.github/workflows/python.yml

```
@@ -0,0 +1,229 @@
+        with:
+          name: wheels
+          path: dist
+
+  release:
+    name: Publish version
+    runs-on: ubuntu-latest
+    environment: production
+    if: github.event_name == 'release'
+    needs: [macos, windows, linux, linux-cross, musllinux, musllinux-cross]
+    steps:
+      - uses: actions/download-artifact@v3
```

actionlint (1.7.8)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)